src/components/common/ModalComponent.jsx

```diff
@@ -60,11 +62,19 @@ ArtModal.propTypes = {
   modalTitle: PropTypes.string,
   trigger: PropTypes.element,
   className: PropTypes.string,
-  modalSize: PropTypes.string
+  modalSize: PropTypes.string,
+  open: PropTypes.bool,
+  closeOnEscape: PropTypes.bool,
+  closeOnDimmerClick: PropTypes.bool,
+  closeIcon: PropTypes.bool
 };
 
 ArtModal.defaultProps = {
   children: <br />,
   modalTitle: '',
-  modalSize: 'small'
+  modalSize: 'small',
+  open: false,
+  closeOnEscape: true,
```

Comment:
Did you try defaulting this to `false`? That is, have `closeOnEscape`, `closeOnDimmerClick` and `closeIcon` default to `false`. This way, you won't have to explicitly make them false in `SessionExpiredComponent.jsx`. The props won't be passed to semantic ui when falsy so this should not affect any existing modal functionality. Please ensure you test the other existing modal's usage to confirm there's no side effects upon making the change.

PS: I wanted to test this but [Semantic UI](https://react.semantic-ui.com/modules/modal/) has a `ReferenceError`.

Reply:
Ok 👍